packages/sandboxd/USER_GUIDE.md

```
@@ -229,6 +229,53 @@ resp, _ := client.Execute(ctx, &processv1.ExecuteRequest{
+The Go and Python SDKs speak sandboxd directly. Because sandboxd binds
+loopback-only inside the pod, the SDKs connect via a **port-forward straight
+to the sandbox pod** (both `:8080` and `:9090`) rather than through the
+sandbox-router — filesystem calls go over REST, `Run` goes over gRPC.
+
+> **Where commands run:** `ProcessService` executes commands **inside the
+> `sandboxd` container**, not the workload container. The base `sandboxd`
+> image is minimal (a shell + coreutils), so `Run` can use `sh`, `cat`,
+> `ls`, etc. out of the box; to run a language runtime (e.g. `python3`),
+> build a sandboxd image that includes it. Files written over REST land in
+> the shared `/workspace` volume, visible to both containers.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)